app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -258,12 +258,26 @@
                     }
                 } ar = 0; } gh = 0; } ci = 0; } l = 0; } c = 0; } g = 0; }
 
-                vm.allSetTiers = Object.keys(set_map).sort().reverse();
-                vm.activesets = vm.allSetTiers[0];
+                var tiers = _.each(_.groupBy(Object.keys(set_map), function(set) {
```

Comment:
There was probably a much better way to do this... -_-. Wanted to preserve the sort order in each group.